app/src/main/res/values/strings.xml

```diff
@@ -174,6 +174,8 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="highlight_line_ending_if_two_or_more_spaces">Highlight spaces at the end of lines containing two or more spaces</string>
     <string name="use_monospace_for_code">Use monospace font for code</string>
     <string name="disable_code_block_highlight">Disable highlighting for code blocks</string>
+    <string name="auto_renumber_ordered_list">Auto-renumber ordered lists</string>
+    <string name="auto_renumber_ordered_list_slowdown">This option may impact app performance.</string>
```

Comment:
Have you really spotted a decline in performance? To be honest, the only point where it would be slow is when pressing enter right? I feel like there is no need to confuse users and just have in description what it does. We can add a performance notice later when we see it's really required.

Reply:
I have not spotted a decline in performance but I use a pixel 3, which is a pretty fast phone. I suppose a user could switch to plaintext if they wanted to disable auto-renumbering. 

I will disable this for now.